src/terra-dev-site/doc/UpgradeGuides.c/v6.0.0.t.doc.md

```diff
@@ -0,0 +1,52 @@
+# v6.0.0 Upgrade Guide
+
+This document will provide information on upgrading from Terra Toolkit `5.0.0` to `6.0.0`.
```

Comment:
Should this say `upgrading from Terra Dev Site` instead of `Terra Toolkit`?

Reply:
Yeah, it should, ALL of the upgrade guides in terra-dev-site made this mistake 😅